events/wc_on_another_world_events.txt

```diff
@@ -0,0 +1,142 @@
+namespace = WCOAW
+#On Another World Events
+
+# Set "On Another World" characters as landless
+character_event = {
+	id = WCOAW.1
+
+	is_triggered_only = yes
+	hide_window = yes
+
+	trigger = {
+		trait = on_another_world
+	}
+
+	immediate = {
+        create_title = {
+            tier = DUKE
+            landless = yes
+			name = "ANOTHER_WORLD_TITLE"
+            holder = THIS
+            culture = THIS
+        }
+        set_defacto_liege = THIS
+        
+        # Remove them from councillor position at start of the game
+        remove_title = job_chancellor
+        remove_title = job_marshal
+        remove_title = job_treasurer
+        remove_title = job_spymaster
+        remove_title = job_spiritual
+	}
+}
+
+# on_avoided_imprison_started_war
+# on_became_imprisoned
+# on_avoided_imprison_fled_country
+# Removes imprisonment when character is on another world
+character_event = {
+	id = WCOAW.102
+
+    is_triggered_only = yes
+    hide_window = yes
+    
+    trigger = {
+        trait = on_another_world
+    }
+    
+    immediate = {
+ 	   if = {
+ 		   limit = { prisoner = yes }
+ 		   imprison = no
+ 	   }
+ 	   FROM = {
+           modifier = opinion_arrest_attempt
```

Comment:
Huh? Is it typo? Opinion adding has different syntax.
Look for `opinion`
https://ck2.paradoxwikis.com/Commands

Reply:
Hrrrh, I'm thinking of what I wanted to do here, because I feel like I don't want to add anything there. My memory is escaping me right now.

Reply:
Thanks for pointing this out, this was completely wrong as-is.

Thus fixed it in the last commit, hopefully, with a `remove_opinion`.